website/src/remark/tabs-plugin.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const visit = require('unist-util-visit');
+
+const transformFirstTabNode = node => [
+  {
+    type: 'jsx',
+    value: `<Tabs groupId="examples">\n<TabItem value="${node.lang}" label="JavaScript">`,
+  },
+  {
+    type: node.type,
+    lang: node.lang,
+    value: node.value,
+  },
```

Comment:
```suggestion
  node,
```

?

(I know nothing about remark plugins, so maybe not 😀)